open-budget-il/upload.py

```diff
@@ -136,11 +155,14 @@ def get_doc_rank(self,item):
         return None
 
     def get_search_query(self, queryString, request):
-        searchQueryParts = []#queryString, "type=%s"%self.KIND]
-        for field, typ in self.FTS_FIELDS:
-            fieldVal = request.get(field)
-            if len(fieldVal) > 0:
-                searchQueryParts.append("%s=%s"%(field, fieldVal))
+        searchQueryParts = []
+        if len(self.FTS_FIELDS) > 0:
+            searchQueryParts = [queryString, "type=%s"%self.CLS.__name__]
```

Comment:
The idea is that the search query for each handler will be limited to the handler type.
This way we can add filtering of specific attributes in each handler e.g. year for budget lines and entity type for entities